WordPress/gutenberg
# Fix center alignment on the dropdown in the categories block

## What?

Add `text-align:center` to the block CSS so that the block can be centered when the "Display as dropdown" option is used.
Fixes https://github.com/WordPress/gutenberg/issues/43982

## Why?

The center alignment was not working in the editor or the front.

## How?

The markup in the editor and front is different, so CSS needed to be added to both style.scss and editor.scss.
The text alignment is limited to when the dropdown option is used, because otherwise, the text inside the list items would be centered, but not the list bullets.

## Testing Instructions
1. In the block editor, add two categories blocks.
2. Center both, and turn on "Display as dropdown" on one of theme.
3. Confirm that both blocks are centered.
4. Save and view the front, confirm that both blocks are centered on the front.

## Screenshots or screencast
